plugins/star-rating/frontend/public/templates/comments-table.html

```diff
@@ -0,0 +1,35 @@
+<div class="comments-table">
+  <cly-datatable-n
```

Comment:
similar idea of wrapping within cly-main applies here too.
its mentioned below.